src/addons/Confirm/Confirm.js

```diff
@@ -52,8 +54,12 @@ class Confirm extends Component {
   };
 
   render() {
+    let classes = classNames(
+      'sd-confirm',
+      this.props.className,
+    );
```

Comment:
We use the `classnames` module to add the `sd-*` class, inherit `this.props.className` and add Semantic UI classes.